docker-compose.yml

```
@@ -9,9 +9,9 @@ services:
     env_file:
       - ./.flaskenv
     environment:
-      - DATABASE_URI=sqlite:////db/map.db
+      - DATABASE_URI=${DATABASE_URI}
     volumes:
       - ./map:/code/map
       - ./db/:/db/
     ports:
-      - "5000:5000"
+      - "${EXTERNAL_PORT}:5000"
```

You may want to include an inline default so `EXTERNAL_PORT` doesn't require explicit configuration
```suggestion
      - "${EXTERNAL_PORT:-8080}:5000"
```